packages/client/components/EditableText.tsx

```diff
@@ -166,7 +166,7 @@ const EditableText = forwardRef((props: Props, ref: any) => {
     const inProps = {
       autoFocus: true,
       ref: inputRef as any,
-      maxLength,
+      maxLength: maxLength + 1,
```

Comment:
-2 This component is used elsewhere, please don't change global components to fix a local issue without retesting all other usages of those components.

Reply:
Hello @Dschoordsch , regarding the general fix in this PR, we figured that the bug mentioned  in this issue seemed to be occurring in other areas where this component was being used as well. For instance, when updating the `organization name`.

Reply:
Can you please update the testing scenarios to the new changes?

Reply:
Hi @Dschoordsch, we are unable to locate any ui tests that require updating on this instance. This recent change only result in a visual difference, without affecting the current unit test suite for the `Legality` file.

Reply:
@gitstart-parabol sorry for not being more clear what I meant: Could you please update the manual test cases mentioned in the description of this PR to also test for regressions in other places where your changes apply?